### DISCUSSION TITLE
Serivice started automatically

### DISCUSSION THREAD

**Discussion by tarcoiz** [dnschneid/crouton#4419] [Q&A]
I'm using Debian 9 (Stretch) on Chromeos with crouton.
I set crouton to auto start at Chromebook boot, and  service(Apahce) to auto start  by insserv, but the Service is not started.
Is it possible to start automatically?

insserv -s output excerpt
> K:01:0 1 6:apache2
> S:02:2 3 4 5:apache2

**Comment by ghost** [comment]
Take a look at [ChromeOS-AutoStart](https://github.com/supechicken/ChromeOS-AutoStart) by @supechicken. It's a Chrome OS extension that runs commands when turning on your device, I haven't tested it with crouton.

**Reply by tarcoiz** [reply]
Thank you for your answer.
Then,  what command should I input to boot crouton?

**Reply by ghost** [reply]
It depends. I would think that if you have a desktop for your chroot, like XFCE, you could just run `sudo startxfce4` (or what ever desktop you use). I don't think it'll work with non-gui chroots. 

Once again I haven't tested it with crouton yet, I am hopping to soon.

**Reply by supechicken** [reply]
@tarcoiz You can use `sudo enter-chroot <command to start the service>` to start it on a chroot without starting a desktop environment :)
